src/state/StateKeyValue.cpp

```diff
@@ -559,7 +559,7 @@ uint32_t StateKeyValue::waitOnRedisRemoteLock(const std::string& redisKey)
             break;
         }
 
-        std::this_thread::sleep_for(std::chrono::milliseconds(1));
+        SLEEP_MS(1);
```

Comment:
I was dealing with so many sleeps and writing `usleep(123 * 1000)` that I made a macro.